server/store/middleware/infohash/blacklist.go

```diff
@@ -13,17 +13,22 @@ import (
 func init() {
 	tracker.RegisterAnnounceMiddleware("infohash_blacklist", blacklistAnnounceInfohash)
 	tracker.RegisterScrapeMiddlewareConstructor("infohash_blacklist", blacklistScrapeInfohash)
+	mustGetStore = func() store.StringStore {
+		return store.MustGetStore()
+	}
 }
 
 // ErrBlockedInfohash is returned by a middleware if any of the infohashes
 // contained in an announce or scrape are disallowed.
 var ErrBlockedInfohash = tracker.ClientError("disallowed infohash")
 
+var mustGetStore func() store.StringStore
```

Comment:
I thought about putting the mock somewhere else, I just don't know where and if it's really useful